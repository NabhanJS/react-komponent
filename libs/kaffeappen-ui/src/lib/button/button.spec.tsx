import { render } from '@testing-library/react';
import Button from './button';

describe("Button", () => {
    it("should test the button", () => {
        const { baseElement } = render(< Button variant={'login'} label={"login"}/>)
        expect(baseElement).toBeTruthy()
    })
    it("should match the snapshot", () => {
        const { baseElement } = render(< Button variant={'login'} label={"login"}/>)
        expect(baseElement).toMatchSnapshot()
    })
})