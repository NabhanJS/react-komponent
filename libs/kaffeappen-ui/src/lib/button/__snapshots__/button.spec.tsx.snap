// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match the snapshot 1`] = `
<body>
  <div>
    <button
      class="button__StyledButton-ox3zwd-0 lmzMDR"
    >
      login
    </button>
  </div>
</body>
`;
